src/cloud/release-notes/mcp-release-notes.md

```diff
@@ -17,30 +17,36 @@ The `{{site.data.var.mcp}}` package uses the following version sequence: `<major
 See [Apply patches]({{site.baseurl}}/cloud/project/project-patch.html) to learn how to apply Magento patches and hot fixes to your {{site.data.var.ece}} project.
 
 ## v1.0.2
-*Release date: February 24, 2020*<br/>
+*Release date: February 27, 2020*<br/>
 
--  {:.fix}**Added the [PayPal Express Checkout issue with region patch for Magento v2.3.4](https://magento.com/tech-resources/download#download2353)**-Published on February 12, 2020,  this patch resolves an issue that affects orders placed with PayPal Express Checkout where the shipping address for the order specifies a country region that has been manually entered into the text field rather than selected from the drop-down menu on the Shipping page. See [Apply patches]({{site.baseurl}}/cloud/project/project-patch.html).
+This release includes the following patches and critical fixes:
 
--  {:.fix}**Updated the [Scope parameter for Async/Bulk API patch](https://magento.com/tech-resources/download#download2312) for Magento Open Source v2.3.1 and 2.3.2**-Fixed a syntax error in the `composer.json` file.
+-  {:.new}**Compatibility updates for `{{site.data.var.mcp}}`**–
```

Comment:
I'm not so sure you need to distinguish New or Fix or anything in the Patch notes. That was to distinguish between a new feature in the main tool. Can probably do away with that convention for these notes. I always expect patches to fix something, not introduce a new feature.